xdggs/h3.py

```diff
@@ -22,6 +21,40 @@
 from xdggs.utils import _extract_cell_id_variable, register_dggs
 
 
+def polygons_shapely(wkb):
+    import shapely
+
+    return shapely.from_wkb(wkb)
+
+
+def polygons_geoarrow(wkb):
+    import pyproj
+    import shapely
+    from arro3.core import list_array
+
+    polygons = shapely.from_wkb(wkb)
+    crs = pyproj.CRS.from_epsg(4326)
+
+    geometry_type, coords, (geom_offsets, ring_offsets) = shapely.to_ragged_array(
+        polygons
+    )
+
+    if geometry_type != "Polygon":
+        raise ValueError(f"unsupported geometry type found: {geometry_type}")
+
+    polygon_array = list_array(geom_offsets, list_array(ring_offsets, coords))
```

Comment:
on my phone but you need a dict with a `crs` key still. So you should use `CRS.to_json_dict`, then wrap that in a python dict and encode to json.

Reply:
or `f'{{"crs": {CRS.to_json()}}}'`, I suppose. Thanks for catching this!